### DISCUSSION TITLE
How to alter ibm logo in footer?

### DISCUSSION THREAD

**Discussion by fiskce** [carbon-design-system/carbon#10487] [Help] [answered]
How to alter ibm logo in footer?

How do i change this? cant find anything about it?

**Comment by tay1orjones** [comment]
@fiskce Could you share more of what you're referencing? Are you using [`gatsby-theme-carbon`](https://github.com/carbon-design-system/gatsby-theme-carbon) perhaps?

**Reply by fiskce** [reply]
yes i am indeed using the gatsby theme. but i dont see a discussion page on the other repo. so i decided to post here.

**Comment by tay1orjones** [comment, marked as answer]
@fiskce Sorry for the horrible delay here. To override pieces of `gatsby-theme-carbon`, you'll need to shadow the targeted component. In this case that should be the Footer. Full info is [available in the docs](https://gatsby-theme-carbon.vercel.app/guides/shadowing).
